### ISSUE TITLE
[SETUP-BUG] localhost refused to connect on GCP notebook

### ISSUE THREAD

**Issue by Robotatron** [voxel51/fiftyone#1468]
Using Jupyter Lab notebooks on a GCP VM instance (Vortex AI notebooks). When executing `session = fo.launch_app(dataset)` the app doesnt start but instead shows **localhost refused to connect**. 

![image](https://user-images.githubusercontent.com/95919873/145578171-88aa6f56-1a67-4fde-a39b-05bd19057dc3.png)

**Comment by Robotatron** [comment]
Any suggestions of what I can try out or how to debug it?

**Comment by benjaminpkane** [comment]
@Robotatron we do not officially support Vortex AI notebooks. Likely only a  small amount of changes need to be made. 

We will have a new release soon that resolves a handful of outstanding notebook improvements. January, likely.

Releated: #1339, #1343, #1389

**Comment by mariaculman18** [comment]
Same problem here. Is a new release available?

**Comment by Robotatron** [comment]
> @Robotatron we do not officially support Vortex AI notebooks. Likely only a small amount of changes need to be made.
> 
> We will have a new release soon that resolves a handful of outstanding notebook improvements. January, likely.
> 
> Releated: #1339, #1343, #1389

Thanks for the reply. Any updated ETA for Vortex AI integration? Just a rough estimate, I understand you have priorities etc.

**Comment by Robotatron** [comment]
@benjaminpkane  I just realized the notebook I am using is a normal GCP cloud instance, a normal VM machine.
Maybe I need to enable port forwarding or something else for the fiftyone UI to work properly?

**Comment by benjaminpkane** [comment]
Yes, directions for remote notebooks can be found [here](https://voxel51.com/docs/fiftyone/environments/index.html#remote-notebooks).

**Comment by Robotatron** [comment]
> Yes, directions for remote notebooks can be found [here](https://voxel51.com/docs/fiftyone/environments/index.html#remote-notebooks).

Remote machine does not seem to work under Windows - https://github.com/voxel51/fiftyone/issues/1732

However, this ticket is not about remote connection, fiftyone simply does not work in a regular notebook when run on Google VM instances. Instead of rendering the UI the cell shows an error `localhost refused connection`

There must be something in the configuration of the VM, maybe port forwarding or permission change to make it work.
VM is a regular machine, it should work same as locally.

**Comment by Robotatron** [comment]
It's hard for me to believe I am the only user in the world who uses Google VM instances + fiftyone, there must be a workaround for that:

![image](https://user-images.githubusercontent.com/95919873/167229082-e976e750-b1fd-4402-ad98-313f5ecad56c.png)

**Comment by benjaminpkane** [comment]
This will be resolved in due time. There are quite a few cloud products that each offer their own versions of "Notebooks". Each one handles networking a little differently.

Any VM can be used with a remote setup, port forwarding just needs to be used so localhost connections work.

The only one we officially support is [Colab](https://colab.research.google.com/github/voxel51/fiftyone-examples/blob/master/examples/quickstart.ipynb), but we will be adding more very soon, including this one (the next couple releases).

**Comment by benjaminpkane** [comment]
@austinmw not officially. But sagemaker is the next environment we want to add. Contributions are welcome

**Comment by hossenkamal** [comment]
>@austinmw not officially. But sagemaker is the next environment we want to add. Contributions are welcome

@benjaminpkane  you mentioned about not officially. Do you have any workaround currently that works for the sagemaker environment ?
